packages/react-components/react-tag-picker/library/src/utils/useExpandLabel.ts

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import { useTagPickerContext_unstable } from '../contexts/TagPickerContext';
+import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
+import { TagPickerControlState } from '../TagPickerControl';
+
+export function useExpandLabel(options: { tagPickerId: string; state: Pick<TagPickerControlState, 'expandIcon'> }) {
+  const { tagPickerId, state } = options;
+  const { targetDocument } = useFluent();
+  const triggerRef = useTagPickerContext_unstable(ctx => ctx.triggerRef);
+  const expandIconRef = React.useRef<HTMLSpanElement>(null);
+
+  const expandIcon = { ...state.expandIcon };
```

Comment:
now that you're not returning the state, and don't need to mutate it, you actually don't need to assign `...state.expandIcon` to a new variable anymore.